app/src/test/java/org/fossasia/openevent/app/core/presenter/EventDashboardPresenterTest.java

```diff
@@ -113,6 +118,8 @@ public void tearDown() {
 
     @Test
     public void shouldLoadEventAndAttendeesAndStatisticsAutomatically() {
+        when(databaseChangeListener.getNotifier()).thenReturn(PublishSubject.create());
```

Comment:
Add this is setUp to avoid writing in each function

Reply:
It is only required in 8/25 tests. That's why i added it explicitly in those methods

Reply:
OK